clicache/src/impl/PdxLocalWriter.hpp

```diff
@@ -83,18 +83,18 @@ namespace Apache
           }
 
           /// <summary>
-          /// Write a byte to the <c>IPdxWriter</c>.
+          /// Write an signed byte to the <c>IPdxWriter</c>.
```

Comment:
'a signed' is correct grammar.  Sorry to be a pain

Reply:
Not a pain. Your diligence is appreciated.

Fixed.